### ISSUE TITLE
Photon Rate slider changes to rate regions, step size, and help text.

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#274]
In last week's design meeting we decided to reduce the described number of rates on the Photon Rate slider from 5 to 4 rate regions and make the default key press 1 region. And in addition change the help text for the slider.

**Described regions for photon rates:**
- light source off (0)
- very slow (1)
- slow (2)
- fast (3)

**Step sizes:**
- Default step size with Arrow keys: 1 region
- Larger step size with Page Up & Page Down keys: same as default (not needed)
- Smallest step size (with Shift key): same as default (not needed)
- Home key: min position (i.e., light source off)
- End key: max position (i.e., fast)

**Change Help Text from:**
- Adjust to emit photons at different rates.

**Comment by jessegreenberg** [comment]
This should be done in https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.25/phet/molecules-and-light_en_phet.html, can you please verify @terracoda?

**Comment by terracoda** [comment]
@jessegreenberg, I see I cut myself off in the first comment (apologies).

Please change Help Text from "Adjust to emit photons at different rates."
TO:
- Set photon emission rate.

**Comment by terracoda** [comment]
@jessegreenberg, please assign reviews to @Matthew-Moore240

**Comment by jessegreenberg** [comment]
OK updated above. @Matthew-Moore240 can you please make sure master matches https://github.com/phetsims/molecules-and-light/issues/274#issue-566487934? Please let me know if you have any questions about testing the latest version on phettest.

**Comment by terracoda** [comment]
his is all good @jessegreenberg!
Thanks!
